src/garage/envs/environment.py

```diff
@@ -0,0 +1,188 @@
+"""Base Garage Environment API."""
+
+import abc
+
+
+class Environment(abc.ABC):
+    """The main API for garage environments.
+
+    The main public API methods are:
```

Comment:
instead of saying "main", I recommend you make this statement unqualified "The public API methods are..." -- to me, "main" implies that there might be more, which makes me anxious as someone who wants to make sure I implement the full API.